app/src/main/java/pl/droidevs/books/entity/Book.java

```diff
@@ -0,0 +1,53 @@
+package pl.droidevs.books.entity;
+
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.PrimaryKey;
+
+import static pl.droidevs.books.entity.Book.TABLE_NAME;
+
+@Entity(tableName = TABLE_NAME)
+public class Book {
```

Comment:
Jeśli zdecydujemy się na dwa różne obiekty: domenowy i bazodanowy to proponowałbym nazwę: BookEntity.